foundation/foundation-mda/src/main/resources/templates/inference/inference.impl.py.vm

```diff
@@ -18,10 +18,13 @@ def load_model():
     utilization for inferencing.
     :return:
     """
+    config = InferenceConfig()
+
     try:
-        config = InferenceConfig()
         # Load the model from the model directory
         # model = mlflow.sklearn.load_model(config.model_directory())
+        
+        pass 
```

Comment:
Q: Should this try catch block be removed?

Reply:
I believe it's best to keep the try-catch block to ensure that if `config.model_directory()` is incorrectly assigned or if mlflow is unable to load the model, then some logging is created. 

The changes I've made resolves the logic issue where if the `config` is in the try block, then `config` can't be used in the except block.